installer/ovms/description.yaml

```diff
@@ -0,0 +1,33 @@
+name: ovms
+product: openvino-model-server
+version: "0.1.0"
+description: |
+  OpenVINO Model Server Kubernetes Operator
+install:
+  - type: kustomize
+    location: kustomize
+    waitfor:
+      - namespace: ovms-operator-system
+        selector: all
+    waitfor:
```

Comment:
you do not need to speciy `waitfor` twice, just use it as a list of steps

Reply:
nice, thanks :)